driverconfig/config.yaml

```diff
@@ -143,6 +143,13 @@ drivers:
         external-health-monitor: gcr.io/k8s-staging-sig-storage/csi-external-health-monitor-controller:v0.4.0
   - name: powerstore
     configVersions:
+      - configVersion: v2.4.0
+        useDefaults: true
+        supportedVersions:
+          - version: v121
+          - version: v122
+          - version: v123
+          - version: v124
       - configVersion: v2.3.0
         useDefaults: true
```

Comment:
This should be set to false and appropriate sidecar versions should be mentioned

Reply:
The side car versions for 2.4.0 and 2.3.0 are same. So it is set to true here. Should we still set false here?

Reply:
As discussed offline, addressed this comment.